.github/workflows/cd.yml

```
@@ -410,54 +75,54 @@ jobs:
           method: "POST"
           customHeaders: |
             Content-Type: application/json
           data: |
             {
-              "text": ":rocket: 배포 성공 - `${{ github.repository }}` 브랜치 `${{ github.ref_name }}`"
+              "content": "🚀 배포 성공 - `${{ github.repository }}` 브랜치 `${{ github.ref_name }}`"
             }
 
-      - name: Notify Mattermost on Deploy Failure
+      - name: Notify Discord on Deploy Failure
         if: failure() && steps.deploy-ec2.conclusion == 'failure'
```

actionlint (1.7.7)

property "deploy-ec2" is not defined in object type {health-check: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)